src/interdiction/Interdiction.jl

```diff
@@ -0,0 +1,249 @@
+__precompile__(true)
+module Interdiction
+
+using LightGraphs
+using JuMP
+using MathProgBase
+
+import MathProgBase.SolverInterface.AbstractMathProgSolver,
+       JuMP.UnsetSolver
+
+export interdiction_flow, ⪷,
+       NetworkInterdictionProblem, AdaptiveFlowArcProblem, AdaptiveFlowPathProblem,
+       MultilinkAttackAlgorithm, BilevelMixedIntegerLinearProgram
```

Comment:
I think you need to export more than this, especially given https://travis-ci.org/JuliaGraphs/LightGraphsExtras.jl/builds/156426906#L2989.